samples/apps/copilot-chat-app/WebApp/src/components/chat/ChatHistoryItem.tsx

```diff
@@ -97,14 +97,16 @@ export const ChatHistoryItem: React.FC<ChatHistoryItemProps> = (props) => {
             time;
     }
 
-    const isMe = message.sender === account?.homeAccountId;
-    const member = chat.getAudienceMemberForId(message.sender, selectedId, conversations[selectedId].audience);
+    // TODO: Ask Tao to use UserId (immutable through course of profile)
+    // standardize to one
+    const isMe = message.userName === account?.name || message.userId === account?.homeAccountId;
```

Comment:
This should be limited to the userId since it's a unique GUID and immutable through the user profile lifespan

Reply:
it might make sense to return a viewmodel with an additional property that has calculated a message.IsMe boolean server-side vs. client-side